portfolio/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>Joshua King</title>
```

Comment:
This change right here updates the text in the browser tab. So now whenever you access the portfolio page in a browser tab, the tab will use your name as the title.